Dockerfile

```diff
@@ -44,94 +54,52 @@ RUN set -eux; \
 	chown redmine:redmine "$HOME"; \
 	chmod 1777 "$HOME"
 
-ARG REDMINE_VERSION="4.2.3"
-ARG REDMICA_VERSION=""
-# ENV REDMINE_DOWNLOAD_SHA256 ad4109c3425f1cfe4c8961f6ae6494c76e20d81ed946caa1e297d9eda13b41b4
+# To support Redmine 5.0
+ENV GEM_RGEO_ACTIVERECORD_VERSION=7.0.1
+ENV GEM_ACTIVERECORD_POSTGIS_ADAPTER_VERSION=7.1.1
```

Comment:
This is also not the big issue, but these gem versions are not necessary in recent GTT plugin version.